src/Sylius/Bundle/CoreBundle/Templating/Helper/PriceHelper.php

```diff
@@ -43,6 +43,32 @@ public function getPrice(ProductVariantInterface $productVariant, array $context
         ;
     }
 
+    /**
+     * {@inheritdoc}
+     *
+     * @throws \InvalidArgumentException
+     */
+    public function getOriginalPrice(ProductVariantInterface $productVariant, array $context): int
+    {
+        Assert::keyExists($context, 'channel');
+
```

Comment:
```suggestion
        Assert::isNotInstanceOf($this->productVariantPriceCalculator, ProductVariantPricesCalculatorInterface::class); 
        
```

Reply:
Same below

Reply:
Didn't you meant :
`Assert::isInstanceOf($this->productVariantPriceCalculator, ProductVariantPricesCalculatorInterface::class); `
?

Reply:
Yes, of course :)